lib/web_console/whiny_request.rb

```diff
@@ -5,8 +5,8 @@ module WebConsole
   # return false, an info log message will be displayed in users' logs.
   class WhinyRequest < SimpleDelegator
     def from_whitelisted_ip?
-      whine_unless request.from_whitelisted_ip? do
-        "Cannot render console from #{request.remote_ip}! " \
+      whine_unless request.from_whitelited_ip? do
```

Comment:
The correctly-spelled name is the right one since #170... which means this should surely be causing a test failure somewhere? :confused: